insights/parsers/samba.py

```diff
@@ -24,4 +24,30 @@ class SambaConfig(IniConfigFile):
 
     def parse_content(self, content):
         lstripped = [line.lstrip() for line in content]
```

Comment:
I think you're better off putting the two 'lstripped' expressions into one here.